app/scripts/modules/Registration.js

```diff
@@ -0,0 +1,40 @@
+var React = require('react');
+
+var Registration = React.createClass({
+  render: function() {
+    return (
+      <form className="registration" role="form" onChange={this.handleChange}>
+        <div>
+          <div>
+            <input className="registration__input" type="text" name="fName" ref="firstName" placeholder="First Name"/>
+          </div>
+        </div>
+        <div>
+          <div>
+            <input className="registration__input" type="text" name="lName" ref="lastName" placeholder="Last Name"/>
```

Comment:
no need for Last name. use 1 field to have both first and last name e.g 'John Doe'